clang/test/CodeGenSYCL/multiple_aspect_enum.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -internal-isystem %S/Inputs -fsycl-is-device -triple spir64-unknown-unknown -fsyntax-only -verify -emit-llvm-only %s
```

Comment:
Is `-emit-llvm-only` needed when `-fsyntax-only` is present?
```suggestion
// RUN: %clang_cc1 -internal-isystem %S/Inputs -fsycl-is-device -triple spir64-unknown-unknown -fsyntax-only -verify %s
```

Reply:
`-emit-llvm-only` is required, but `-fsyntax-only` does not seem to be. It seems like most tests using `-emit-llvm-only` do not use `-fsyntax-only`, with a few exceptions. I will remove `-fsyntax-only`.